README.md

```diff
@@ -87,22 +87,22 @@ sudo rm /usr/local/bin/eosio-*
 * eosio-objdump
 * eosio-readelf
 
-Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided_installation_or_building_from_scratch)
-eosio-abidiff
-eosio-ranlib
-eosio-ar
-eosio-objdump
-eosio-readelf
+Below tools are not installed after brew install, you get them only by building the repository and installing from scracth, [see here](#guided-installation-or-building-from-scratch)
+* eosio-abidiff
+* eosio-ranlib
+* eosio-ar
+* eosio-objdump
+* eosio-readelf
```

Comment:
Also a careful read of the PRs (from draft release notes shows that this is not the case moving forward) Please see https://github.com/EOSIO/eosio.cdt/pull/810

Reply:
A similar issue exists for LLVM 7 mention earlier in this file - now set to LLVM 9 https://github.com/EOSIO/eosio.cdt/pull/896

Reply:
Fixed.